reportportal/oci/create-junit-file.py

```
@@ -0,0 +1,73 @@
+#!/usr/bin/env python3
+import sys
+import json
+import json
```

Ruff (0.14.10)

Redefinition of unused `json` from line 3

Remove definition: `json`

(F811)